app/Repositories/RoleRepository.php

```diff
@@ -10,4 +10,14 @@ public function model()
     {
         return Role::class;
     }
+
+    public function pluckRole()
+    {
+        return $this->model->pluck('display_name', 'id');
+    }
+
+    public function getIdTrainee()
+    {
+    	return $this->model->select('id')->where('name', 'like', 'trainee')->first();
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.DisallowTabIndent.TabsUsed</br>**Severity**: error</br>**File**: app/Repositories/RoleRepository.php</br>**Line**: 21</br>Spaces must be used to indent lines; tabs are not allowed